docs/radicle.md

```diff
@@ -4,13 +4,17 @@ title: What is Radicle?
 sidebar_label: What is Radicle?
 ---
 
-Radicle is a peer-to-peer stack for code collaboration 🌱. It enables developers
+Radicle is a set of protocols design for decentralized code collaboration 🌱. It enables developers
 to collaborate on code without relying on trusted intermediaries. Radicle was designed
 to provide similar functionality to centralized code collaboration platforms — or "forges" —
 while retaining Git’s peer-to-peer nature, building on what made distributed
 version control so powerful in the first place.
 
-Read more about our vision for decentralized code collaboration [here][wh].
+<blockquote>
+Read more about our vision for decentralized code collaboration <a href="https://radicle.xyz/blog/towards-decentralized-code-collaboration.html">here</a>.
+</blockquote>
```

Comment:
You can use `>` to introduce block quotes in markdown
```suggestion
> Read more about our vision for decentralized code collaboration [here][wh].
```

Reply:
😍 Lifesaver!